apps/browser/src/tools/popup/send/send-add-edit.component.ts

```diff
@@ -124,7 +124,10 @@ export class SendAddEditComponent extends BaseAddEditComponent {
 
   cancel() {
     // If true, the window was pop'd out on the add-send page. location.back will not work
-    if ((window as any).previousPopupUrl.startsWith("/add-send")) {
+    if (
+      (window as any).previousPopupUrl == null ||
+      !(window as any).previousPopupUrl.startsWith("/add-send")
+    ) {
```

Comment:
🎨 This can be simplified using nullish-resolution:

```suggestion
    const isPopup = (window as any)?.previousPopupUrl?.startsWith("/add-send") ?? false;
    if (!isPopup) {
```